misc/db_schema.yaml

```diff
@@ -1615,20 +1608,20 @@ users:
     - { Field: email, Type: varchar(64), 'Null': false, Extra: '' }
     - { Field: descr, Type: char(30), 'Null': false, Extra: '' }
     - { Field: level, Type: tinyint(4), 'Null': false, Extra: '', Default: '0' }
-    - { Field: can_modify_passwd, Type: tinyint(4), 'Null': false, Extra: '', Default: '1' }
-    - { Field: created_at, Type: timestamp, 'Null': false, Extra: '', Default: '1970-01-02 00:00:01' }
+    - { Field: can_modify_passwd, Type: tinyint(1), 'Null': false, Extra: '', Default: '1' }
+    - { Field: created_at, Type: timestamp, 'Null': false, Extra: '', Default: CURRENT_TIMESTAMP }
```

Comment:
I'll need to check this but I'm sure you can't have two CURRENT_TIMESTAMPs in one table with strict mode on

Reply:
Could be, I just checked and I thought I was running in strict, but I was not.

Reply:
```
260 -> 261 ...SQL Error! SQLSTATE[HY000]: General error: 1293 Incorrect table definition; there can be only one TIMESTAMP column with CURRENT_TIMESTAMP in DEFAULT or ON UPDATE clause (SQL: ALTER TABLE `users` CHANGE `created_at` `created_at` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ;)
  /opt/librenms/includes/sql-schema/update.php:92
```

Reply:
I think only one could have the `on update CURRENT_TIMESTAMP` right?  default CURRENT_TIMESTAMP is ok?

Reply:
No it says you can't for default in that output